web/themes/custom/nysenate_theme/src/templates/views/views-view-fields--committee-meetings--agenda.html.twig

```diff
@@ -39,6 +39,7 @@
 {% set abstained_count = fields.field_ol_abstained_count.content|striptags|trim is not empty ? fields.field_ol_abstained_count.content : 0 %}
 {% set absent_count = fields.field_ol_absent_count.content|striptags|trim is not empty ? fields.field_ol_absent_count.content : 0 %}
 
+{% if aye_count %}
```

Comment:
Is your intention to disable this entire block if there are no "AYE" votes?  IOW, you want to display nothing if there were votes, but they were all "NAY" or "ABSTAIN"?

Reply:
that was left over from just testing a few things out seeing what might work ... i forgot about it, i just removed it

Reply:
well look at that.. peer review works.  😏